apps/zipsync/src/__snapshots__/start.test.ts.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`CLI Tool Tests should display help for "zipsync --help" 1`] = `
 "
-zipsync 0.0.0 - https://rushstack.io
+zipsync 0.1.0 - https://rushstack.io
```

Comment:
This is going to keep causing problems with every version bump. You may need to override this with Jest mocking, or use string replacement. How is it handled in Rush?

Reply:
updated test to replace the version with `{version}` string.